src/site/stages/build/process-cms-exports/transformers/node-event.js

```diff
@@ -0,0 +1,123 @@
+const {
+  getDrupalValue,
+  utcToEpochTime,
+  getWysiwygString,
+  uriToUrl,
+} = require('./helpers');
+const { mapKeys, camelCase } = require('lodash');
+const assert = require('assert');
+
+function fakeUtc(timeString) {
```

Comment:
I have no idea if this is right, but I have no source timezone, so :man_shrugging: 

I could also use `moment-timezone`, but figured that may end up being more work than just swapping out a `T` for a space and adding ` UTC`. :man_shrugging: